### PR TITLE
disable download links on entry view

### PR DESCRIPTION
**Pull request by nicosomb**
only because export is still not yet implemented.

### CODE REVIEW

**Comment by tcitworld** [comment]
`<strike>` is depreciated with html5. Can you use `<del>` instead ?

**Comment by nicosomb** [comment]
Sure I can.

**Comment by nicosomb** [comment]
done

File: src/Wallabag/CoreBundle/Resources/views/themes/material/Entry/entry.html.twig [hunk 1]
```diff
@@ -83,13 +83,13 @@
         <li class="bold">
             <a class="waves-effect collapsible-header">
                 <i class="mdi-file-file-download small"></i>
-                <span>{% trans %}Download{% endtrans %}</span>
-            </a>
+                <span><del>{% trans %}Download{% endtrans %}</del></span>
+                </a>
```
Comment (j0k3r) [inline src/Wallabag/CoreBundle/Resources/views/themes/material/Entry/entry.html.twig]: extra useless tabulation? :)

**Comment by nicosomb** [comment]
Fixed.

**Comment by j0k3r** [comment]
Aaaaand now you can squash everything so we can merge it!

**Comment by nicosomb** [comment]
squash

**Comment by j0k3r** [comment]
:+1: :laughing:

**Pull request merged**